torch/onnx/symbolic_opset9.py

```diff
@@ -2286,6 +2287,7 @@ def batch_norm(
         return res
 
 
+@_beartype.beartype
```

Comment:
You may remove the beartype check for this one here. I can handle it with #84091